templates/bin_setup

```diff
@@ -13,8 +13,12 @@ bundle check || bundle install
 # Set up database and add any development seed data
 bin/rails dev:prime
 
-# Add binstubs to PATH via export PATH=".git/safe/../../bin:$PATH" in ~/.zshenv
-mkdir -p .git/safe
+if [ ! -d .git/safe ] && echo $PATH | grep .git/safe > /dev/null; then
+  echo "-----------------------------------------------------------------------"
+  echo
+  echo "-> When you trust this repo, remember to run: mkdir -p .git/safe"
```

Comment:
Looks like we lost part of the help text from the previous comment?

```
Add binstubs to PATH via export PATH=".git/safe/../../bin:$PATH" in ~/.zshenv
```

Would be nice to drop the Z shell-specific part of that instruction.

The instruction "When you trust this repo, remember to run: mkdir -p .git/safe" doesn't fully accomplish the goal of using the project's binstubs unless they've also configured their `PATH`, though?

Worth adding to this instruction something like this?

> Add this project's binstubs to your PATH via export PATH=".git/safe/../../bin:$PATH"

Or we write the be-all, end-all blog post about this whole topic and then link to that blog post from this instruction?